packages/avatar/src/lib/Avatar.tsx

```diff
@@ -0,0 +1,48 @@
+import { Avatar as MuiAvatar, AvatarProps as MuiAvatarProps } from '@mui/material';
+import React from 'react';
+import { tokens } from '@availity/design-tokens';
+
+export interface AvatarProps extends Omit<MuiAvatarProps, 'variant'> {
+  children: React.ReactNode;
+  /** The size of the component.
+   * @default xl */
+  size?: 'xs' | 's' | 'm' | 'l' | 'xl';
+}
+
+const getInitials = (name: string, size: 'xs' | 's' | 'm' | 'l' | 'xl'): string => {
+  if (name.split(' ').length > 1 && size !== 'xs') {
+    return `${name.split(' ')[0][0]}${name.split(' ')[1][0]}`;
+  }
+
+  return name.charAt(0);
+};
+
+const formatChildren = (children: React.ReactNode, size: 'xs' | 's' | 'm' | 'l' | 'xl'): React.ReactNode => {
+  if (typeof children === 'string') {
+    return getInitials(children, size).toUpperCase();
+  }
+
+  return children;
+};
+
+export const Avatar = ({ children, size = 'xl', sx, ...rest }: AvatarProps): JSX.Element => {
+  const sizeStyling = {
```

Comment:
Since these are static I think we should move the definition outside the component

Reply:
good point. done